### ISSUE TITLE
correct handling of symbols with same name

### ISSUE THREAD

**Issue by jonorthwash** [jonorthwash/ud-annotatrix#14]
Currently (at least in CG mode), symbols with the same name screw things up.  For example, if you have two `<,>` nodes, you need to rename them to something unique, like `<,1>` and `<,2>` for the tree to display correctly.  This should be fixed.

**Comment by SilentFlame** [comment]
@jonorthwash can you please give me some sentence trees with which I can check how does it really works, and give a try to resolving the issue.

Some link where I could find some sample example sentence trees of the related formats would also be helpful.

**Comment by jonorthwash** [comment]
Here's a good sentence with three commas:
```
"<Патшамен>"
        "патша" n ins @nmod #1->3
"<соғыс>"
        "соғыс" n nom @obj #2->3
"<ашқанда>"
        "аш" v tv ger_past loc @advcl #3->12
"<,>"
        "," cm @punct #4->12
"<ел-жұрт>"
        "ел-жұрт" n nom @conj #5->7
"<,>"
        "," cm @punct #6->7
"<отанымды>"
        "отан" n px1sg acc @obj #7->8
"<қорғауға>"
        "қорға" v tv ger dat @advcl #8->12
"<,>"
        "," cm @punct #9->12
"<біз>"
        "біз" prn pers p1 pl nom @nsubj #10->12
"<соғысқа>"
        "соғыс" n dat @nmod #11->12
"<бардық>"
        "бар" v iv ifi p1 pl @root #12->0
"<.>"
        "." sent @punct #13->12
```

**Comment by jonorthwash** [comment]
Here's one with a number of like-named nodes (and you can make it worse by lower-casing the first letter of the first node):
```
"<Біреу>"
        "біреу" prn ind nom @nsubj #1->5
"<қазақ>"
        "қазақ" n nom @nsubj #2->3
"<жақсы>"
        "жақсы" adj @ccomp #3->5
                "е" cop aor p3 sg @cop #4->3
"<деді>"
        "де" v tv ifi p3 sg @root #5->0
"<,>"
        "," cm @punct #6->5
"<біреу>"
        "біреу" prn ind nom @nsubj #7->11
"<орыс>"
        "орыс" n nom @nsubj #8->9
"<жақсы>"
        "жақсы" adj @ccomp #9->11
                "е" cop aor p3 sg @cop #10->9
"<деді>"
        "де" v tv ifi p3 sg @parataxis #11->5
"<.>"
        "." sent @punct #12->11
```

**Comment by SilentFlame** [comment]
@jonorthwash That means we have to do the same for any symbol which is repeated.(i.e number them so as to consider them different) am I right?

**Comment by jonorthwash** [comment]
No, @SilentFlame.  The solution for this will probably just involve tracking tokens in a slightly different data structure—currently there can only be one token of a given form.  We don't want to impose restrictions like this on the data; instead, we want to adapt the software to deal with a wider range of data.

**Comment by jonorthwash** [comment]
Probably fixed by commit 20b0f115f30249847afd2649b016da7a3b9e27ad ; need to test.

**Comment by jonorthwash** [comment]
Tested; works!
